idb/postgres/postgres.go

```diff
@@ -2433,8 +2433,8 @@ func (db *IndexerDb) GetAccounts(ctx context.Context, opts idb.AccountQueryOptio
 
 	if opts.HasAssetID != 0 {
 		opts.IncludeAssetHoldings = true
-	} else if (opts.AssetGT != 0) || (opts.AssetLT != 0) {
-		err := fmt.Errorf("AssetGT=%d, AssetLT=%d, but HasAssetID=%d", opts.AssetGT, opts.AssetLT, opts.HasAssetID)
+	} else if (opts.AssetGT != nil) || (opts.AssetLT != nil) {
+		err := fmt.Errorf("AssetGT=%d, AssetLT=%d, but HasAssetID=%d", *opts.AssetGT, *opts.AssetLT, opts.HasAssetID)
```

Comment:
There's a possible nil deference here since there are two variables. You can grab this helper to safely deference them:
```
func uintOrDefault(x *uint64) uint64 {    
  if x != nil {    
    return *x    
  }    
  return 0    
}    
```

Reply:
good catch